jest.config.js

```diff
@@ -1 +1,4 @@
 module.exports = require('@jupiterone/integration-sdk-dev-tools/config/jest');
+const path = require('path');
+
+require('dotenv').config({ path: path.resolve(__dirname, './.env') });
```

Comment:
Let's move to this pattern: https://github.com/JupiterOne/integration-template/issues/29

Reply:
Cool. Done.